src/app/medInria/areas/workspaces/visualization/medVisualizationWorkspace.h

```diff
@@ -23,12 +22,11 @@ class medVisualizationWorkspace : public medAbstractWorkspaceLegacy
                             "Workspace to visualise images, meshes and other data types.",
                             "Basic")
 public:
-     medVisualizationWorkspace(QWidget *parent = 0);
-    ~medVisualizationWorkspace();
-
-    virtual void setupTabbedViewContainer();
+    medVisualizationWorkspace(QWidget *parent = nullptr): medAbstractWorkspaceLegacy(parent){}
 
     static bool isUsable();
```

Comment:
It's static and return always true ...
Suicide candidate ?

Reply:
I did not change that part, but it's a mandatory function used in medWorkspaceFactory -> registerWorkspace. Indeed it's always true because since there is no toolbox in this special workspace, it would hide it. 
Do you want to do something in that part? Since it's unchanged code, it may be done in an other PR perhaps?

Reply:
Yes will be done something in other PR.